src/main/java/com/dongtronic/diabot/platforms/discord/listeners/HelpListener.kt

```diff
@@ -1,19 +1,33 @@
 package com.dongtronic.diabot.platforms.discord.listeners
 
-import com.dongtronic.diabot.platforms.discord.commands.DiabotCommand
 import com.dongtronic.diabot.exceptions.NoCommandFoundException
+import com.dongtronic.diabot.platforms.discord.commands.DiabotCommand
 import com.jagrosh.jdautilities.command.Command
 import com.jagrosh.jdautilities.command.CommandEvent
 import net.dv8tion.jda.api.EmbedBuilder
 import net.dv8tion.jda.api.Permission
 import net.dv8tion.jda.api.entities.ChannelType
+import net.dv8tion.jda.api.exceptions.ErrorResponseException
 import net.dv8tion.jda.api.exceptions.InsufficientPermissionException
+import net.dv8tion.jda.api.requests.ErrorResponse
 import java.awt.Color
-import java.security.Permissions
 import java.util.*
 import java.util.function.Consumer
 
 class HelpListener : Consumer<CommandEvent> {
+    /**
+     * Executed when the attempt to send a DM to a user fails
+     */
+    private fun sendingError(exc: Throwable, event: CommandEvent) {
+        if (exc is ErrorResponseException
+                && exc.errorResponse != ErrorResponse.CANNOT_SEND_TO_USER) {
+            // Print stack trace if the error code was not related to DMs being blocked
+            exc.printStackTrace()
+        }
+
+        event.replyError("Could not send you a DM, please adjust your privacy settings to allow DMs from server members.")
```

Comment:
"[...] or send me a DM first"

I think that would work too, right?

Reply:
Not from what I tested, Clyde responds with the following message when you DM the bot with privacy settings set to prevent DMs from server members.

>  Your message could not be delivered because you don't share a server with the recipient or you disabled direct messages on your shared server, recipient is only accepting direct messages from friends, or you were blocked by the recipient.

Reply:
Right, yeah makes sense.